mathesar_ui/src/component-library/tabs/TabContainer.scss

```diff
@@ -22,25 +22,26 @@
       user-select: none;
       list-style: none;
       overflow: hidden;
-      transition: width 0.2s cubic-bezier(0.39, 0.575, 0.565, 1);
+      transition: border 0.2s ease-in-out,
+        width 0.2s cubic-bezier(0.39, 0.575, 0.565, 1);
       border-radius: 2px 2px 0px 0px;
       border-bottom: 2px solid transparent;
       margin-bottom: -1px;
+      font-size: var(--text-size-large);
 
       > div {
-        padding: 0.5rem 1rem;
+        padding: 0.75rem 0;
         white-space: nowrap;
         text-decoration: none;
         text-overflow: ellipsis;
         overflow: hidden;
         color: inherit;
-        text-align: center;
-        font-weight: var(--tab-header-font-weight, 590);
+        font-weight: var(--tab-header-font-weight, 500);
         width: 100%;
       }
 
       &:hover {
-        background: var(--slate-100);
+        //background: var(--slate-100);
```

Comment:
I notice a lot of commented out styles. Please remove them.

Reply:
I'd like these changes in this PR itself.

Reply:
I have just merged this branch into my work in #2185, and of the conflict resolution involved removing commented-out code. **Please don't make any more changes within `/src/systems/record-selector/`** (even to delete commented-out code). Leaving your PR as-is within that directory will be fine because I've removed the commented-out code in my branch. Removing the commented-out code here will require me to resolve conflicts again.